src/smc_sagews/smc_sagews/sage_jupyter.py

```diff
@@ -84,6 +84,7 @@ def _get_doc(self):
     __doc__ = property(_get_doc)
 
 jupyter = JUPYTER()
+octave = jupyter('octave')
```

Comment:
I'm a bit puzzled about what you try to do here. Trying to understand your intentions, this line means that each time a sagews starts, octave starts up too? That doesn't sound like a good idea to me.